M3U8.md

```diff
@@ -83,14 +83,6 @@ Legend for playlist types:
 IETF drafts notes
 -----------------
 
-[IETF](http://ietf.org) document currently in Draft status. Different versions of the document introduce changes of HLS protocol playlist formats. Latest version of the HLS protocol is version 7.
```

Comment:
I don't agree on this.
There is an RFC and there is an Internet Draft, so talking about one state is wrong.